deploy/template.yaml

```diff
@@ -47,34 +47,45 @@ Conditions:
 Mappings:
   EnvironmentConfiguration:
     dev:
+      logLevel: "debug"
       dynatraceSecretArn: arn:aws:secretsmanager:eu-west-2:216552277552:secret:DynatraceNonProductionVariables
+      fargateCPUsize: "1024"
+      fargateRAMsize: "2048"
+      desiredTaskCount: 2
+      ga4Enabled: "true"
     build:
+      logLevel: "debug"
```

Comment:
Can we make this info level as were going to perf test in build and dont want to flood the logs.
Can we also make this the same as prod for cpu and ram as we will perf test on build